docs/spec.md

```diff
@@ -132,6 +139,66 @@ Informs the client that channels are no longer available.
 }
 ```
 
+### Parameter Values
+
+This message is sent as a response to the client's [Get Parameters request](#get-parameters). Only supported if the server declares the `parameters` [capability](#server-info).
```

Comment:
Can the server choose to send parameters on its own or does the client have to request them?

Reply:
With the rosbridge protocol, the client has to actively request paramters (via a service call), that's why I opted for that mechanism. But you are right, I see no reason why the server can not also publish available parameters without being requested to do so. If we allow this, I will delete the first sentence of this paragraph